src/tests/ftest/nvme/enospace.py

```diff
@@ -100,8 +101,13 @@ def ior_bg_thread(self):
 
         # Define the job manager for the IOR command
         job_manager = get_job_manager(self, "Mpirun", ior_bg_cmd, mpi_type="mpich")
-        self.create_cont()
-        job_manager.job.dfs_cont.update(self.container.uuid)
+
+        # create container
+        container = TestContainer(self.pool, daos_command=DaosCommand(self.bin))
+        container.get_params(self)
+        container.create()
```

Comment:
I'd recommend using this existing function:
https://github.com/daos-stack/daos/blob/2b0d4a638977a4f8c7a0566ea9a8160ed0bcd573/src/tests/ftest/util/apricot/apricot/test.py#L1669
```suggestion
        # create container
        container = self.get_container(self.pool)
```